content/iOS/RSFormView/RSFormViewExample.md

```diff
@@ -0,0 +1,310 @@
+                                 initialValue: "",
+                                 placeholder: "FIRST NAME",
+                                 fieldType: .regular,
+                                 isValid: false,
+                                 errorMessage: "First name can't be empty")
+  let lastNameField = FormField(name: "LAST NAME",
+                                initialValue: "",
+                                placeholder: "LAST NAME",
+                                fieldType: .regular,
+                                isValid: false,
+                                errorMessage: "Last name can't be empty")
+
+  return FormItem(firstField: firstNameField, secondField: lastNameField)
+}()
+
+lazy var personalInfoHeaderItem: FormItem = {
+  let headerItem = FormItem()
+  headerItem.attributedText = NSAttributedString(string: "Enter your personal info",
+                                                 attributes: [.foregroundColor: UIColor.brightGray,
+                                                              .font: UIFont.systemFont(ofSize: 20)])
+  return headerItem
+}()
+
+func init() {
+  items = [nameItem, personalInfoHeaderItem]
+}
+```
+
+Here we are only adding two items, to see the whole implementation you can download the example project I'm attaching at the end of this article.  
+Since we are setting two FormFields in the first item it will be rendered as a cell with two text fields. 
+For the header we are only setting a `NSAttributedString` so it will be rendered as a single label cell with the appearance we set in the attributes. 
+
+Now we go back to our FormExampleViewController and we define a var to hold our form helper. 
+
+``` swift 
+var formHelper = ExampleFormHelper()
+```
+
+Finally, we define actions for the submitButton and modeSwitch: 
+
+``` swift
+//MASK: Actions
+@objc
+func switchValueChanged(sender: UISwitch) {
+  let isDarkMode = sender.isOn
+  let configurator = isDarkMode ? DarkModeConfigurator() : FormConfigurator()
+  formHelper.updateHeaders(isDarkMode: isDarkMode)
+  formView.formConfigurator = configurator
+  view.backgroundColor = isDarkMode ? UIColor.mineShaftGray : UIColor.white
+  descriptionLabel.textColor = isDarkMode ? UIColor.white : UIColor.brightGray
+  switchLabel.textColor = isDarkMode ? UIColor.white : UIColor.brightGray
+}
+
+@objc
+func submitButtonTapped() {
+  var collectedData = ""
+  for field in formHelper.fields() {
+    collectedData += "\(field.name): \(field.value) \n"
+  }
+  let alert = UIAlertController(title: "Collected data",
+                                message: collectedData,
+                                preferredStyle: .alert)
+  alert.addAction(UIAlertAction(title: "OK",
+                                style: .cancel))
+  present(alert, animated: true)
+}
+```
+`switchValueChanged` changes our views aspect according to the selected value, if on we go for a "dark mode" otherwise we go for a clear mode. 
```

Comment:
It's kind of hard to understand the 'if on' refers to the switch state at first glance. I would rather use: "if set to on"